packages/manager/apps/dedicated/client/app/account/user/components/newAccountForm/translations/Messages_fr_CA.json

```diff
@@ -288,7 +288,7 @@
   "signup_enum_country_PY": "Paraguay",
   "signup_enum_country_QA": "Qatar",
   "signup_enum_country_QC": "Quebec",
-  "signup_enum_country_RE": "Réunion, Îles",
```

Comment:
Same content is also available here:
https://github.com/ovh/manager/blob/master/packages/manager/modules/sign-up/src/form/translations/Messages_fr_FR.json#L193

Out of curiosity, is there any reason to not prefer using `Réunion` instead?
Because it seems to be already used at multiple places (see links below). 

- https://github.com/ovh/manager/blob/master/packages/manager/modules/sign-up/src/form/translations/Messages_fr_FR.json#L193
- https://github.com/ovh/manager/blob/master/packages/manager/apps/telecom/src/app/common/translations/Messages_fr_FR.json#L413
- https://github.com/ovh/manager/blob/master/packages/manager/apps/web/client/app/core/translations/Messages_fr_FR.json#L440

According to `iso.org`, using `Réunion` seems to be correct.
Source: https://www.iso.org/obp/ui/fr/#iso:code:3166:RE

Reply:
Bonjour Antoine,
Comme je suis à l'origine de la demande de correction proposée ici par @chtitux https://twitter.com/f1oren/status/1424236255605248001 et pour répondre à la question, l'usage actuel sur l'île, et recommandé par  la Commission nationale de toponomie française, est d'utiliser « La Réunion  » avec une majuscule à l'article http://sig974.free.fr/?p=565

Contrairement par ex. à la Martinique, l'article fait partie du nom La Réunion. En aucune façon la mention « Île » ou «  Île de  » n'en fait partie.

Pouvez-vous corriger aux différents endroits dans l'interface ? Au choix « Réunion, La » ou « La Réunion »

Merci d'avance
Florent

Reply:
Thanks a lot @f1oren for feedback.

According to your inputs, I've updated all occurrences in our code base in order to make sure that wherever we rely on `country_RE`, it says `La Réunion`.

@chtitux please do not pay attention to my previous "Request changes" review, I've completed the branch by adding a new commit.

Waiting an Internal approval before being able to go further. I will keep you posted.

Reply:
Don't worry, thanks a lot for your help and contribution!

Reply:
Thanks a lot @chtitux and @antleblanc